Additional tests files added

Srajald opened the pull request:
here are the test cases for --
test_run_subprocess.py
test_simple_workflow.py

jcharkow commented on tests/test_simple_workflow.py:
```diff
@@ -0,0 +1,79 @@
+    df = launch.dataframe[0].value
+    assert df.shape == (5, 4), f"Expected table size (5,4) but got {df.shape}"
+
+def test_invalid_inputs(launch):
+    """Ensure invalid inputs are handled properly."""
+
+    launch.run(timeout=10)
+    time.sleep(2) 
+
+    x_input = next((ni for ni in launch.number_input if ni.key == "example-x-dimension"), None)
+    y_input = next((ni for ni in launch.number_input if ni.key == "example-y-dimension"), None)
+
+    assert x_input is not None, "X-dimension input not found!"
+    assert y_input is not None, "Y-dimension input not found!"
+
+
```
Please see this suggestion above. I would try testing an x value greater than 20 and it should be invalid.

jcharkow reviewed (changes requested):
Good progress so far! Please see comments above. 

Some additional comments:
1. It does not seem that sample.mzML is used so please remove that file. 
2.  These tests are currently not triggered by github actions. I would recommend adding these functions to test_gui.py instead of adding new files. Alternatively if you think it is necessary to add new files please edit .github/ci.yml so these files are also tested

jcharkow reviewed (changes requested):
Seems that tests are still failing see comment above.

jcharkow reviewed (changes requested):
Looks good just some very minor changes.

Srajald commented:
@jcharkow 
I have Updated that sir,
I have added that just for the precaution measure..
rest now it is removed and ready to be merges..
Thanks Sir..

jcharkow reviewed (approved):
Looks good! Ready to merge
